### ISSUE TITLE
disable accessibility by default on macOS

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#2477]
Re: https://github.com/rstudio/rstudio/issues/1990

Some users have bumped into this in the wild. We should disable accessibility features by default to avoid these issues, but provide UI so that users can re-enable it if necessary.

(Note that RStudio is already currently not accessible: users with screen readers are unable to view the contents of an Ace editor; however, I believe that may be changing in a future release of Ace)

**Comment by kevinushey** [comment]
Oops -- looks like we already do this! https://github.com/rstudio/rstudio/blob/8799b9508112bdd1e67da126f5bc47e9e544ba23/src/cpp/desktop/DesktopMain.cpp#L267-L274

https://github.com/rstudio/rstudio/issues/2183 tracks the need for UI to enable / disable this.
